manifests/server/config.pp

```diff
@@ -23,11 +23,15 @@
 
   if $logbin {
     $logbindir = dirname($logbin)
-    file { $logbindir:
-      ensure => directory,
-      mode   => '0755',
-      owner  => $options['mysqld']['user'],
-      group  => $options['mysqld']['user'],
+    
+    #Stop puppet from managing directory if just a filename/prefix is specified
+    if $logbindir != '.' {
```

Comment:
i'm confused about this: when do we (anyone?) set logbindir to .? how do we know to set it to .?